.github/workflows/release-promote.yml

```diff
@@ -320,8 +320,7 @@ jobs:
           branch-name: '${{ steps.release_branch.outputs.BRANCH_NAME }}'
           pr-title: 'chore(release): bump version to ${{ needs.calculate-versions.outputs.NEXT_NIGHTLY_VERSION }}'
           pr-body: 'Automated version bump to prepare for the next nightly release.'
-          app-id: '${{ secrets.APP_ID }}'
-          private-key: '${{ secrets.PRIVATE_KEY }}'
+          github-token: '${{ secrets.GEMINI_CLI_ROBOT_GITHUB_PAT }}'
```

Comment:
Sanity check: I'm reading about Github PAT tokens and it seems that they can be scoped to specific ACLs and rights.  Just confirming that the ACLs here are tuned for "goldilocks just right", and that we don't have excessive permissions here.

As a tangent, once we stabilize changes on all of this release pipeline stuffs, I think we should join forces and write up some docs around all this stuff.  I feel like we both have a little bit of knowledge that will got lost in the sands of time if we don't jot it down.  I'm absolutely happy to participate